### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by crtmneric**
Adds missing steps for installation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,6 +17,14 @@ Then, to enable this plugin, run::
 
     tutor plugins enable notes
 
+Then, save config::
+
+    tutor config save
```
Comment (regisb) [inline README.rst:22]: There is no need to run `config save` because it is part of `local launch`.

**Review by regisb — CHANGES_REQUESTED** [review]
Thanks for this PR! You're right that the instructions do not include `local launch`. That's because this instruction should be the same for all plugins -- though it's not made very clear in the official docs. Also, users who run `tutor dev` or `tutor k8s` will have to run a different command.

File: README.rst (hunk 1 above)
Comment (crtmneric) [reply README.rst]: Thanks for the feedback removed unnecessary step.

**Review by regisb — APPROVED** [review]
thanks for the PR!

**Pull request merged**